ocaml/idl/datamodel_certificate.ml

```diff
@@ -69,10 +69,10 @@ let t =
             [(Published, rel_stockholm, ""); (Deprecated, "24.19.0", "")]
           ~ty:String "fingerprint" ~default_value:(Some (VString ""))
           "Use fingerprint_sha256 instead"
-      ; field ~qualifier:StaticRO ~lifecycle ~ty:String "fingerprint_sha256"
+      ; field ~qualifier:RW ~lifecycle:[] ~ty:String "fingerprint_sha256"
```

Comment:
Please keep this read only, clients are not supposed to change the value

Reply:
Ah okay sorry, I thought it just needed RW if it was ever changed.